packages/tofu/src/findGlobals.js

```diff
@@ -15,6 +15,17 @@ const nonReferenceIdentifiers = [
   'RestElement',
 ]
 
+const importExportSpecifierTypes = new Set(
+  /** @type {const} */ ([
+    'ImportSpecifier',
+    'ImportDefaultSpecifier',
+    'ImportNamespaceSpecifier',
+    'ExportSpecifier',
+    'ExportDefaultSpecifier',
+    'MetaProperty',
```

Comment:
this is `import.meta`, which we need to account for insofar as we need to tell tofu it isn't a "global"